### PR TITLE
Move written explanation to required section

### PR DESCRIPTION
**Pull request by kylelrichards11**
I think that this requirement could be reworded as well. Right now it seems like it's really important to have visualizations and discuss those, when in reality it's really important to have a written explanation with some visualizations. If you have any thoughts on a better wording for this bullet, I think that could be helpful as well.

### CODE REVIEW

File: data-engineering/README.md [hunk 1]
```diff
@@ -11,12 +11,12 @@ Your objectives are:
 
 - Map the equipment data from SQL into arrays (or a single array). To keep things simple, you can ignore the static data to start. Be sure to describe your design choices and the array schema you chose.
 - Apply filters to clean the data. For example, there are some clear outliers in the data due to communication errors from the sensor equipment. These bad measurements have no bearing on the ability for Tagup to provide insight from the data, and a good place to start is to find a way to filter them out.
```
Comment (sjmccorm1993) [inline data-engineering/README.md:13]: I would remove the "For example" in this sentence

File: data-engineering/README.md [hunk 2]
```diff
@@ -11,12 +11,12 @@ Your objectives are:
...
+- Demonstrate the efficacy of your approach using visualizations. You must also include a simple explanation of these figures and why your approach is effective, written in language that non-technical executives could understand.
 
 Bonus points:
 - Provide summary statistics for the ingressed data, including various statistical moments, and any other relevant descriptive statistics.
 - Integrate the static data into your design.
```
Comment (sjmccorm1993) [inline data-engineering/README.md:18]: Is there a reason not to make this required? A clever way of capturing both static and timeseries data is a pretty good litmus test for what we're trying to measure

**Review by sjmccorm1993 — APPROVED** [review]
LGTM

File: data-engineering/README.md (hunk 2 above)
Comment (kylelrichards11) [reply data-engineering/README.md:18]: I potentially agree, but maybe we should remove `To keep things simple, you can ignore the static data to start.` from the first bullet point? But I would rather a candidate do a good job with the timeseries data than get stumped with a good way to include everything.

Comment (sjmccorm1993) [reply data-engineering/README.md:18]: yeah, I see your point. How about this compromise: make integrating the static data the first thing listed under bonus points?

**Pull request merged**